### DISCUSSION TITLE
Bromite keeps crashing

### DISCUSSION THREAD

**Discussion by NineOutOfTen** [bromite/bromite#1894] [General]
After the last update, the browser crashes when I try to open it. Unfortunately, there have been occasions when Bromite's adblocker wouldn't work and now this.

Please fix this and make sure your updates don't result in problems. This is frustrating to say the least.

Also, please have a feature put in place to allow for users to add their own search engines.

**Comment by ghost** [comment]
> the browser crashes

I do not face this... try reinstalling Bromite 

> here have been occasions when Bromite's adblocker wouldn't work and now this.

Bromite ad Blocker can't block some ads which are delivered by the first party site not ad domains (e.g. Youtube)

> have a feature put in place to allow for users to add their own search engines.

Bromite already does check [this wiki page](https://github.com/bromite/bromite/wiki/searchengines).

**Reply by NineOutOfTen** [reply]
I did remove and reinstall the browser 3 times before posting this.

**Reply by anuraganand-15** [reply]
I have the same issue the browser crashes as soon as i launch it.
I suspect this is happening due to changes made to notify users for outdated version.
It happened to me 2 times whenever a new build was up while i was on an older version.
Try upgrading to the new version it fixed the problem for me both times.

Hello @csagan5  can you guide me on how to get a crashlog for the same.
My device - OnePlus Nord
Android version - 11
Bromite builds which crashed for me -v99.0.4844.58
v99.0.4844.55

**Reply by Beach77** [reply]
I am also having this problem second time now. Last time I fixed it by downloading the apk and installing it. This time I can't do that because it says failed to install. I cannot reinstall the app because I have too many bookmarks that I don't want to lose. I need another fix.

**Reply by 1voltpp** [reply]
It crashes right after opening on 2 devices. Google Pixel 6 and Xiaomi note 10. Both using latest Android and bromite version. Tried several reinstalls.

**Reply by anuraganand-15** [reply]
Try fresh install of a new version.
It worked in my case.

**Reply by Beach77** [reply]
I actually did fix mine. The reason for it failing to install was because I had too little space.

**Comment by ghost** [comment]
Update : Bromite has started sometimes crashing in background (For Me) too.

Bromite Version :- `99.0.4844.77`
Android Versiom :- `8.1`

**Comment by csagan5** [comment]
If you experience a crash with latest version then please submit an issue and attach either the crash dump or the relevant `adb logcat` lines.
